### PR TITLE
Regenerate route manifest on page changes

### PR DESCRIPTION
**Pull request by beerose**
Closes: https://github.com/blitz-js/blitz/issues/3343

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `canary` branch](https://github.com/blitz-js/blitzjs.com/tree/canary))

### CODE REVIEW

**Review by flybayer — COMMENTED** [review]
Otherwise looks good! Is the watcher automatically killed when the server is killed?

**Comment by beerose** [comment]
> Otherwise looks good! Is the watcher automatically killed when the server is killed?

@flybayer That's something I'm not sure about. Do you know the best way to test it?

File: packages/blitz-next/package.json [hunk 1]
```diff
@@ -8,8 +8,7 @@
     "lint": "eslint . --fix",
     "test": "vitest run",
     "test-watch": "vitest",
-    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
-    "postinstall": "node scripts/postinstall.js"
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist"
```
Comment (beerose) [inline packages/blitz-next/package.json:11]:
```suggestion
    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
    "postinstall": "node scripts/postinstall.js"
```

**Comment by dillonraphael** [comment]
Working PR/build https://github.com/blitz-js/blitz/pull/3385

**Pull request closed without merging**